common/source/docs/common-rc-systems.rst

```diff
@@ -169,7 +170,7 @@ Note 2: See :ref:`common-frsky-yaapu`. The ability to change parameters over FRS
 
 Note 3: ArduPilot provides a means to send its telemetry data via CRSF such that it can be displayed on `OpenTX <https://www.open-tx.org/>`__ transmitters using the :ref:`Yaapu Telemetry LUA Script<common-frsky-yaapu>`. The ability to change parameters over CRSF telemetry from an Open TX compatible transmitter in addition to displaying the telemetry data is also possible. See :ref:`common-crsf-telemetry`
 
-Note 4: ELRS (EpressLRS) is a system that uses the CRSF (TBS Crossfire) RC protocol with several minimizations to simplify the system. It has reduced features but it connects to ArduPilot just like CRSF, when CRSF RXs are attached using a full UART, instead of SBUS protocol to communicate to ArduPilot. See `ExpressLRS site <https://www.expresslrs.org/2.0/>` for more information.
+Note 4: ELRS (ExpressLRS) is a flexible open-source system that can output CRSF, SBUS, or Mavlink (with embedded RC) protocols. Telemetry requires the use of CRSF or Mavlink, and the receiver must be wired to a full UART. ELRS Mavlink can also be operated in :ref:`GCS only mode<common-gcs-only-operation>`. See `ExpressLRS site <https://www.expresslrs.org/>` for more information.
```

Comment:
I think the MAVLink option ONLY can operate as a GCS only setup....

Reply:
That is not correct. Mavlink can operate both as a normal RC link from the TX with a wired or wireless connection to a GCS, or it can operate in GCS only mode (e.g. ELRS TX connected directly to GCS via USB).

Reply:
see note on #6300